infra/recipes/docker-compose/oh-hadoop-spark/cluster.yaml

```diff
@@ -4,6 +4,12 @@ cluster:
     type: "hadoop"
     uri: "hdfs://namenode:9000/"
     root-path: "/data/openhouse"
+  storages:
```

Comment:
During cutover we would have both configs in cluster yaml?
Would it mean we have two Hadoop file IO instances being minted?
Which instance would be used?

Reply:
yes! you are right. During cutover we'll have both configs in cluster yaml and both fileIO instances minted.

The qualifier `LegacyFileIO` would ensure the old fileIO instances is getting used, until we refactor.